tools/docker/image-pull.sh

```diff
@@ -6,13 +6,14 @@
 # See LICENSE file for more details.
 ###############################################################
 
-scriptdir="$(cd "${0%/*}"; pwd)"
-rootdir="${scriptdir%/*/*}"
+scriptdir=$(dirname "$(readlink -f "$0")")
```

Comment:
Again, please use
```suggestion
scriptdir="$(cd "${0%/*}" && pwd)"
```

Reply:
I followed shellcheck advice
```
    In tools/docker/image-pull.sh line 9:
    scriptdir="$(cd "${0%/*}"; pwd)"
                 ^----------^ SC2164: Use 'cd ... || exit' or 'cd ... || return' in case cd fails.
    
    Did you mean:
    scriptdir="$(cd "${0%/*}" || exit; pwd)"
```

Reply:
Well, I don't know why shellcheck advises that, because it's completely equivalent to
```
scriptdir="$(cd "${0%/*}" && pwd)"
```
(which is what we use in most other places that get it right).

Anyway, doesn't matter much.